Src/BootCamp.Chapter/CaesarCipher.cs

```diff
@@ -10,39 +9,29 @@ public static class CaesarCipher
     {
         public static string Encrypt(string message, byte shift)
         {
-            return string.IsNullOrEmpty(message) ? message : EncryptedMessage(message,shift); 
+            return string.IsNullOrEmpty(message) ? message : EncryptedMessage(message, shift, true); 
         }
 
-        private static string EncryptedMessage(string message, byte shift)
+        private static string EncryptedMessage(string message, byte shift, bool shiftRight)
```

Comment:
No no 😄 
Have both decrypt and encrypt, no need an if statement.
But decrypt should reuse encrypt, that's all.

Reply:
Ok :)
Hopefully now I got it right?

Reply:
Yes